examples/parseNumbers.php

```diff
@@ -19,7 +19,7 @@
 $client = new MyTgClientDebug();
 /* @noinspection PhpUnhandledExceptionInspection */
 $client->infoLogin();
-$client->parseNumbers($numbers, false, false, function (array $models) {
```

Comment:
Pls add to commit message that you change default behavior of parseNumbers to downloading photos.